packages/cojson/src/tests/sync.test.ts

```diff
@@ -1656,6 +1652,10 @@ describe("SyncManager - knownStates vs optimisticKnownStates", () => {
       return push.call(jazzCloudConnectionAsPeer.outgoing, msg);
     });
 
+    map.set("key2", "value2", "trusting");
```

Comment:
Had to move the new-content-creating bit after the mock, or it wouldn't work.

I'm sleepy so let me know if this still makes sense @gdorsi